.vscode/launch.json

```diff
@@ -15,6 +15,27 @@
             ],
             "console": "integratedTerminal",
         },
+        {
+            "name": "Attach debugger",
+            "type": "debugpy",
+            "request": "attach",
+            "justMyCode": false,
+            "connect": {
+                "host": "0.0.0.0",
```

Comment:
Can we make this "localhost" for consistency (I've checked it and it still connects)

Reply:
It is no longer necessary to include these in this change as the the launch config will be defined in the workspace file going forward